Added IEEE 1284 Device ID for a Dymo device.

Patch-Origin: Fedora 21

--- a/ppdc/sample.drv
+++ b/ppdc/sample.drv
@@ -125,6 +125,7 @@ Version "1.5"
 {
   Manufacturer "Dymo"
   ModelName "Label Printer"
+  Attribute "1284DeviceID" "" "MFG:DYMO;MDL:LabelWriter 400;"
   Attribute NickName "" "Dymo Label Printer"
   PCFileName "dymo.ppd"
   DriverType label
